tracer/test/test-applications/integrations/Samples.AWS.SimpleNotificationService/Samples.AWS.SimpleNotificationService.csproj

```
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="$(ApiVersion)" />
+    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.7.200.52" />
```

You have to keep this as `$(ApiVersion)` otherwise we're not testing against multiple versions in the CI

```suggestion
    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="$(ApiVersion)" />
```

---

Whoops, I had it before and I rewrote it, nice catch!